WORKSPACE

```diff
@@ -201,11 +201,93 @@ filegroup(
 )
 
 http_archive(
+    name = "ctuning-ai",
+    build_file_content = """
+filegroup(
```

Comment:
Good point. I can't indent the inline `build_file_content` because Starlark is whitespace-sensitive, but I can pull the definition out into another file.